app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -294,7 +299,10 @@ class ProfileEditActivityTest {
       )
     ).use {
       onView(withId(R.id.profile_delete_button)).perform(click())
-      onView(withText(R.string.profile_edit_delete_dialog_positive)).perform(click())
+      onView(withText(context.getString(R.string.profile_edit_delete_dialog_positive)))
```

Comment:
Why isn't ``withText(stringResId)`` sufficient here & below?

Reply:
Yes, `withText(stringResId)` is sufficient here & below. Actually, I followed some other PRs and made this change. Will update soon. Thanks! : )

Reply:
Resolving this on behalf of @BenHenning